docs/developers-guide/build.md

```diff
@@ -90,24 +95,22 @@ Usage:
 Build Sysbox with:
 
 ```
-$ make sysbox
+$ make sysbox-static
 ```
 
-This target creates a temporary container and builds the binaries for the Sysbox
-components inside that container. The resulting binaries are then placed in the
-sysbox-fs, sysbox-mgr, and sysbox-runc sub-directories.
+This creates a temporary "build container" and builds the binaries for the Sysbox
+components inside that container. This way the host is not polluted in any way.
 
-Once you've built Sysbox, you install it with:
+The resulting binaries are then placed in the sysbox-fs, sysbox-mgr, and
+sysbox-runc sub-directories. You can install them on your host with:
 
 ```
 $ sudo make install
 ```
 
-This last target simply copies the Sysbox binaries to your machine's
-`/usr/bin` directory; we don't have a package installer for Sysbox
-(unlike the Sysbox version distributed by Nestybox).
+This copies the Sysbox binaries to your machine's `/usr/bin` directory.
 
-## Cross-compilation
+## Cross-compilation (only needed when building for ARM64 on X86)
```

Comment:
This section is relevant also when building x86 artifacts on an arm64 platform, so please eliminate this new comment.

Reply:
Ok.